tasks/src/test/java/taf/WebDriverYandexDiskAuthTest.java

```diff
@@ -0,0 +1,66 @@
+package taf;
+
+import static taf.constants.Constants.*;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import taf.page.YandexDiskAuthPage;
+import taf.service.YandexDiskService;
+
+public class WebDriverYandexDiskAuthTest {
+  private WebDriver driver;
+
+  @DataProvider(name = "Authentication is valid")
+  public static Object[][] validCredentials() {
+    return new Object[][]{{LOGIN, PASSWORD}};
```

Comment:
It doesn't make sense to use a data provider with only one data set. You can join several data providers to the common one. The provider can contain two data sets: valid and invalid credentials